### PR TITLE
Fixing tribunals api url app setting

### PR DESCRIPTION
**Pull request by t0nin0s**
### Change description ###
Fixing app settings

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: infrastructure/prod.tfvars [hunk 1]
```diff
@@ -2,6 +2,7 @@ infrastructure_env = "prod"
 node_environment = "prod"
 idam_url = "https://hmcts-access.service.gov.uk"
 idam_api_url = "https://idam-api.platform.hmcts.net"
+tribunals_api_url = "http://sscs-tribunals-api-aat.service.core-compute-aat.internal"
```
Comment (chrisdavecm) [inline infrastructure/prod.tfvars]: This looks wrong

File: kubernetes/deployment.template.yaml [hunk 2]
```diff
@@ -23,6 +23,8 @@ spec:
           value: "https://idam-web-public.aat.platform.hmcts.net"
         - name: IDAM_API_URL
           value: "https://idam-api.aat.platform.hmcts.net"
+        - name: TRIBUNALS_API_URL
+          value: "http://sscs-tribunals-api-aat.service.core-compute-aat.internal"
```
Comment (chrisdavecm) [inline kubernetes/deployment.template.yaml]: We don't use this any more, just the charts

Comment (t0nin0s) [reply kubernetes/deployment.template.yaml]: thx, removed!

File: infrastructure/prod.tfvars (hunk 1 above)
Comment (t0nin0s) [reply infrastructure/prod.tfvars]: done

**Pull request merged**